src/errorMsgs.js

```diff
@@ -125,7 +161,37 @@ const findErrors = (function () {
     return checkSyntax;
 }());
 
-
+const tableDirectives = [
+       'api'
+      ,'audit','auditcols',//'audit cols','audit columns'
+      ,'check'
+      ,'colprefix'
+      ,'compress','compressed'
+      ,'insert'
+      ,'rest'
+      ,'select'
```

Comment:
You still have select table directive here which is not inline with you stating you nuked it above.